README.md

```diff
@@ -1,38 +1,66 @@
-# soda-github-action
+# Soda GitHub Action
 [![GitHub Super-Linter](https://github.com/sodadata/soda-github-action/actions/workflows/linter.yml/badge.svg)](https://github.com/marketplace/actions/super-linter)
 [![.github/workflows/tests.yaml](https://github.com/sodadata/soda-github-action/actions/workflows/tests.yaml/badge.svg)](https://github.com/sodadata/soda-github-action/actions/workflows/tests.yaml)
 
-Prevent downstream data quality issues by integrating the Soda Library into your CI/CD pipeline.
+**Soda** enables Data Engineers to test data for quality where and when they need to. It works by taking the data quality checks that you prepare and using them to run a scan of datasets in a data source. 
+A scan is a CLI command which instructs Soda to prepare optimized SQL queries that execute data quality checks on your data source to find invalid, missing, or unexpected data. When checks fail, they surface bad-quality data and present check results that help you investigate and address quality issues.
 
-## Example usage
+Add the **GitHub Action for Soda** to your GitHub Workflow to automatically execute scans for data quality during development.
+
+In a repository in which are adding a transformation or making changes to a dbt model, for example, you can add the Soda GitHub Action to your workflow so that with each new PR, or commit to an existing PR, it executes a Soda scan for data quality and presents the results of the scan in a comment in the pull request, and in a report in Soda Cloud.
+
+Where the scan results indicate an issue with data quality, Soda notifies you both in the PR comment, and by email so that you can investigate and address any issues before merging the PR into production.
+
+Refer to [Soda documentation](https://docs.soda.io/soda/quick-start-dev.html) for an example use case.
+
+## Use the Soda GitHub Action
+
+Add the action to your GitHub Workflow, as in the following example in the `Perform Soda Scan` step.
 
 ```yaml
-- name: Soda scan
```

Comment:
I would leave this snippet as simple example and put the full pipeline example below. In most cases i would assume, users already have a pipeline, they just need the actual action code only.

Reply:
Sure, sounds good to me. Want to strip away the non-essential? Or shall I?

Reply:
Go for it, im off to sleep now :P

Reply:
I left it as-is actually, it is ok. Just removed the python part.